components/zcash_protocol/src/value.rs

```diff
@@ -32,25 +33,25 @@ impl ZatBalance {
 
     /// Creates a constant ZatBalance from an i64.
     ///
-    /// Panics: if the amount is outside the range `{-MAX_MONEY..MAX_MONEY}`.
+    /// Panics: if the amount is outside the range `{-MAX_BALANCE..MAX_BALANCE}`.
     pub const fn const_from_i64(amount: i64) -> Self {
-        assert!(-MAX_MONEY <= amount && amount <= MAX_MONEY); // contains is not const
+        assert!(-MAX_BALANCE <= amount && amount <= MAX_BALANCE); // contains is not const
         ZatBalance(amount)
     }
 
     /// Creates a constant ZatBalance from a u64.
     ///
-    /// Panics: if the amount is outside the range `{0..MAX_MONEY}`.
-    const fn const_from_u64(amount: u64) -> Self {
-        assert!(amount <= (MAX_MONEY as u64)); // contains is not const
+    /// Panics: if the amount is outside the range `{0..MAX_BALANCE}`.
+    pub const fn const_from_u64(amount: u64) -> Self {
```

Comment:
Add to changelog.

Reply:
`ZatBalance` is new, so this isn't a separate addition.